app/views/contexts/summary/group.py

```diff
@@ -28,7 +28,11 @@ def _build_blocks(
         block_ids_on_path = [location.block_id for location in path]
 
         for block in group_schema['blocks']:
-            if block['id'] in block_ids_on_path and block['type'] == 'Question':
+            if (
+                block['id'] in block_ids_on_path
+                and block['type'] == 'Question'
+                and not block.get('hide_on_section_summary', False)
```

Comment:
Enjoyable, 99% of the time this will default and then satisfy the condition :smiley: